src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        final int maxRandomNumber = 100;
```

Comment:
The value '100' is a magic number here. According to the checklist, all magic numbers in your code should be constants. Please define it as a constant field with a meaningful name at the top of the class. Refer to the checklist item about magic numbers for more details.